integration-test-groups/jdbc/oracle/src/main/resources/application.properties

```diff
@@ -20,5 +20,7 @@
 #
 quarkus.datasource.oracle.db-kind=oracle
 quarkus.datasource.oracle.jdbc.max-size=8
-
+# TODO: Remove image override when issues in 23.3-slim are fixed
+# https://github.com/apache/camel-quarkus/issues/5353
+quarkus.datasource.oracle.devservices.image-name=gvenzl/oracle-free:23.2-slim
```

Comment:
Is there any similar issue on quarkus side?

Reply:
I don't see anything yet. I'll wait a day or two to see if a response / fix arrives in the `gvenzl/oci-oracle-free` before reporting it.

Reply:
Seems the Quarkus folks are having the same problem https://github.com/quarkusio/quarkus/pull/36126.

Reply:
created this: https://github.com/gvenzl/oci-oracle-xe/issues/227